concrete-core/src/specification/engines/plaintext_vector_decoding.rs

```diff
@@ -0,0 +1,44 @@
+use super::engine_error;
+use crate::specification::engines::AbstractEngine;
+use crate::specification::entities::{
+    CleartextVectorEntity, EncoderVectorEntity, PlaintextVectorEntity,
+};
+
+engine_error! {
+    PlaintextVectorDecodingError for PlaintextVectorDecodingEngine @
+}
```

Comment:
Check if the encoder vector and the plaintext vector have the same size.

Reply:
Good catch !